_includes/dc/setup.html

```diff
@@ -1,3 +1,18 @@
+{% comment %}
+Astronomy workshops
+{% endcomment %}
+
+
+{% if site.curriculum == "dc-astronomy" %}
+
+<p>
+  The setup instructions for the Data Carpentry Astronomy workshops
+  can be found at <a href="https://datacarpentry.org/astronomy-python/setup.html">
+    the workshop overview site</a>.
+</p>
+
+{% endif %}
```

Comment:
This tag should be removed and line 22 should be `{% elsif site.curriculum == "dc-ecology" %}`

Reply:
```suggestion
```